PKGBUILD

```diff
@@ -1,31 +1,41 @@
 # Maintainer: Thor77 <thor77 at thor77 dot org>
+# Contributor: Sebastian Krebs <sebastian[at]krebs[dot]one>
 
 pkgname=telegraf-bin
 pkgver=1.1.2
 pkgrel=1
```

Comment:
Nah, you didn't. It was included in your patch but i did it on purpose to do it during merge :)